### ISSUE TITLE
Heading has non-empty accessible name - Failed Example 8

### ISSUE THREAD

**Issue by giacomo-petri** [auto-wcag/auto-wcag#1897]
In [Heading has non-empty accessible name](https://www.w3.org/WAI/standards-guidelines/act/rules/ffd0e9/proposed/), [Failed Example 8](https://w3.org/WAI/content-assets/wcag-act-rules/testcases/ffd0e9/0bf7d49ddf99066b816fe42e5cd827a15c7ad24d.html)

> This h1 element has an [explicit role](https://www.w3.org/WAI/standards-guidelines/act/rules/ffd0e9/proposed/#explicit-role) of none. However, the [global](https://www.w3.org/TR/wai-aria-1.1/#global_states) [property](https://www.w3.org/TR/wai-aria/#dfn-property) aria-label is specified. 

Per [ARIA 1.3 - presentation role](https://w3c.github.io/aria/#presentation), the aria-label attribute is not allowed on generic/none/presentation roles.

> Thus it has a [semantic role](https://www.w3.org/WAI/standards-guidelines/act/rules/ffd0e9/proposed/#semantic-role) of heading due to [Presentational Roles Conflict Resolution](https://www.w3.org/TR/wai-aria-1.1/#conflict_resolution_presentation_none). It has an empty [accessible name](https://www.w3.org/WAI/standards-guidelines/act/rules/ffd0e9/proposed/#accessible-name) given by its aria-label attribute and the lack of accessible content to fallback to.

Per [ARIA 1.3 - global states and properties](https://w3c.github.io/aria/#global_states), aria-label is a global state except where prohibited and (again), per [ARIA 1.3 - presentation role](https://w3c.github.io/aria/#presentation) it's prohibited.

Should we remove this failed example from the list?

Giacomo

**Comment by Jym77** [comment]
We're still following ARIA 1.1 in which this example is OK.

---
But indeed, this is not OK anymore in ARIA 1.2.
`aria-label` has become prohibited on `presentation` in ARIA 1.2.

In turn, this means it is not really considered [global](https://www.w3.org/TR/wai-aria-1.2/#global_states) for it anymore:
> The following global states and properties are supported by all roles and by all base markup elements **unless otherwise prohibited**.

(emphasise mine)

Which in turn means that it does not trigger the [presentational role conflict resolution](https://www.w3.org/TR/wai-aria-1.2/#conflict_resolution_presentation_none) anymore:
> If an element has global WAI-ARIA states or properties, user agents MUST ignore the presentation role and expose the element with its implicit role. 

Since `aria-label` is not really global anymore, then the conflict does not triggers…

---
I suggest we let the ARIA 1.2 committee handle that.
This may actually affect quite a few rules where we have examples like this 😖 (and notably #1853 )

**Comment by Jym77** [comment]
https://github.com/w3c/aria/issues/1824

**Comment by Jym77** [comment]
Closing it in light of https://github.com/w3c/aria/issues/1824, the prohibited attribute should still trigger the conflict resolution.
